package.json

```diff
@@ -8,13 +8,15 @@
   },
   "scripts": {
     "clean": "rm -rf build",
-    "prepublish": "npm run build",
+    "prepublishOnly": "npm run build",
```

Comment:
This is better as a `prepare`, that way it also gets run on an npm install via git